engine/installation/linux/oracle.md

```diff
@@ -1,204 +1,236 @@
+    > URL to `testing`. Do not use unstable versions of Docker in production
+    > or for non-testing workloads.
 
-### Use the btrfs storage engine
+2.  Install Docker, changing the path below to the path where you downloaded
+    the Docker package.
 
-Docker on Oracle Linux 6 and 7 supports the use of the btrfs storage engine.
-Before enabling btrfs support, ensure that `/var/lib/docker` is stored on a
-btrfs-based filesystem. Review [Chapter
-5](http://docs.oracle.com/cd/E37670_01/E37355/html/ol_btrfs.html) of the [Oracle
-Linux Administrator's Solution
-Guide](http://docs.oracle.com/cd/E37670_01/E37355/html/index.html) for details
-on how to create and mount btrfs filesystems.
+    ```bash
+    $ sudo yum install /path/to/package.rpm
+    ```
 
-To enable btrfs support on Oracle Linux:
+    The Docker daemon starts automatically.
 
-1. Ensure that `/var/lib/docker` is on a btrfs filesystem.
+3.  Verify that `docker` is installed correctly by running the `hello-world`
+    image.
 
-2. Edit `/etc/sysconfig/docker` and add `-s btrfs` to the `OTHER_ARGS` field.
+    ```bash
+    $ sudo docker run hello-world
+    ```
 
-3. Restart the Docker daemon:
+    This command downloads a test image and runs it in a container. When the
+    container runs, it prints an informational message and exits.
 
-## Uninstallation
+Docker is installed and running. You need to use `sudo` to run Docker commands.
+Continue to [Post-installation steps for Linux](linux-postinstall.md) to allow
+non-privileged users to run Docker commands and for other optional configuration
+steps.
 
-To uninstall the Docker package:
+#### Upgrade Docker
 
-    $ sudo yum -y remove docker-engine
+To upgrade Docker, download the newer package file and repeat the
+[installation procedure](#install-from-a-package), using `yum -y upgrade`
+instead of `yum -y install`, and pointing to the new file.
 
-The above command will not remove images, containers, volumes, or user created
-configuration files on your host. If you wish to delete all images, containers,
-and volumes run the following command:
 
-    $ rm -rf /var/lib/docker
+## Uninstall Docker
 
-You must delete the user created configuration files manually.
+1.  Uninstall the Docker package:
 
-## Known issues
+    ```bash
+    $ sudo yum remove docker-engine
+    ```
 
-### Docker unmounts btrfs filesystem on shutdown
-If you're running Docker using the btrfs storage engine and you stop the Docker
-service, it will unmount the btrfs filesystem during the shutdown process. You
-should ensure the filesystem is mounted properly prior to restarting the Docker
-service.
+2.  Images, containers, volumes, or customized configuration files on your host
+    are not automatically removed. To delete all images, containers, and
+    volumes:
 
-On Oracle Linux 7, you can use a `systemd.mount` definition and modify the
-Docker `systemd.service` to depend on the btrfs mount defined in systemd.
+    ```bash
+    $ sudo rm -rf /var/lib/docker
```

Comment:
This won't work when the `btrfs` graph driver has been used, as the `rm -rf` command cannot remove the subvolumes that Docker creates.